packages/typegpu/src/core/function/tgpuFn.ts

```diff
@@ -259,4 +259,8 @@ class FnCall<Args extends AnyWgslData[], Return extends AnyWgslData | undefined>
       `${ctx.resolve(this._fn)}(${this._params.map((param) => ctx.resolve(param)).join(', ')})`,
     );
   }
+
+  toString() {
+    return `fn call: ${this.label ?? '<unnamed>'}`;
```

Comment:
Imo could be made a bit more inline with the rest of the labels.
```suggestion
    return `call:${this.label ?? '<unnamed>'}`;
```